artifact.sh

```diff
@@ -4,4 +4,4 @@ wget -q -O artifact_go.sh 'https://raw.githubusercontent.com/mdblp/tools/dblp/ar
 chmod +x artifact_go.sh
 
 . ./version.sh
-./artifact_go.sh
+./artifact_go.sh go
```

Comment:
the go-common mongo config is mainly used to build the connection string based on the params (hosts,port,user,password...). And it can be initialized with the connection string. 

But in the main tide-whisperer package, it is initialized with the method FromEnv() that takes separate environments. So we now need them to run it. 

They just use this shortcut for unit testing the store package.